xml/System.Formats.Tar/TarEntryType.xml

```diff
@@ -14,15 +14,7 @@
   </Base>
   <Docs>
     <summary>Specifies the tar entry types.</summary>
-    <remarks>
-      <format type="text/markdown"><![CDATA[
-
-## Remarks
-
-Tar entries with a metadata entry type aren't exposed to the user. They are handled internally.
-
-      ]]></format>
-    </remarks>
+    <remarks>Tar entries with a metadata entry type aren't exposed to the user. They are handled internally.</remarks>
```

Comment:
Is this info appropriate to put in docs?

Reply:
Yes. The "ExtendedAttributes", "LongLink" and "LongPath" entries represent metadata. The user won't see TarEntry instances of this kind when reading an archive, but we added them to stay aligned with the spec.